pyndustri.pyi

```diff
@@ -413,14 +413,31 @@ class Unit(Senseable):
 
         found, x = Unit.locate('enemy', building='core')  # the x coordinate of the enemy core if found
         found, x, y = Unit.locate('enemy', building='core')  # the x and y coordinate of the enemy core if found
-        found, x, y, building = Unit.locate('enemy', building='core')  # the x and y coordinate of the enemy core if found, and specific building type
+        found, x, y, building = Unit.locate('enemy', building='core')  # the x and y coordinate of the enemy core if found, and specific building
 
         Exactly one keyword argument is allowed.
 
         Building can be any literal of: core, storage, generator, turret, factory, repair, rally (command center), battery, resupply, reactor.
         Ore can be one of those in `Env` or in a variable.
         Both spawn and damaged can only be set to `True`.
         """
+    @staticmethod
+    def build(self, x: int, y: int, block_type, rotation=0, config=0):  # build
+        """
+        Build block at position x,y with rotation (0-360 degrees).
+        Block represents the type of the block: (Env.router, Env.sorter, etc.).
+        Config is configuration for block (e.g. Env.titanium for Env.sorter).
+        You can pass onother block variable as config to copy the configuration of that block.
```

Comment:
I've tried the build command with the [6.0 Build 126.2 release](https://github.com/Anuken/Mindustry/releases/tag/v126.2) and the values for rotation seem to range between 0-3. In the beta from Steam values are correctly wrapped around (`rotation % 4`), but in this stable release, it seems to bug out (at least in sandbox) and never finish building.

```suggestion
        Build a block at position `(x, y)` with rotation (0, 1, 2 or 3, the values indicating the number of 90º counter-clockwise steps).
        For example, conveyors start facing east with a rotation of 0, north with 1, west with 2, and south with 3.

        Block represents the type of the block (e.g. Env.router, Env.sorter...).
        Config is configuration for block (e.g. Env.titanium for Env.sorter).
        You can pass another block variable as config to copy the configuration of that block.
```